atom/browser/media/media_stream_devices_controller.cc

```diff
@@ -133,40 +133,45 @@ void MediaStreamDevicesController::Accept() {
         }
         break;
       }
-      case content::MEDIA_DEVICE_ACCESS:
+      case blink::MEDIA_DEVICE_ACCESS: {
         // Get the default devices for the request.
         MediaCaptureDevicesDispatcher::GetInstance()->GetDefaultDevices(
             microphone_requested_, webcam_requested_, &devices);
         break;
+      }
+      case blink::MEDIA_DEVICE_UPDATE: {
+        NOTREACHED();
```

Comment:
should we do something else with this event?

Reply:
It was added in https://chromium-review.googlesource.com/c/chromium/src/+/1354442. The feature seems specifically designed for chrome's screen sharing UI, can consider it for improvement in our desktop capture model once the issue goes public and has a bit more context.